client/src/i18n/fr/form.json

```diff
@@ -680,26 +680,26 @@
             "FILTER_NAME": "Filtrer par Nom...",
             "FILTER_BY": "Filtrer par",
             "SEARCH_NAME": "Chercher Par Nom",
-            "SEARCH_PATIENT_ID": "Chercher ID Patient",
+            "SEARCH_PATIENT_ID": "Chercher par l'ID",
             "QUANTITY": "Entrer le Quantité"
         },
         "SELECT": {
             "EMPTY": "Vide",
             "EMPLOYEE": "Sélectionner un Employé",
             "ACCOUNT": "Sélectionner un compte",
-            "ACCOUNT_TYPE": "Sélectionner Type de Compte",
+            "ACCOUNT_TYPE": "Sélectionner un Type de Compte",
             "ACCOUNT_CONFIGURATION" : "Sélectionner la configuration des comptes",
             "BALANCE_SECTION": "Sélectionner la section du bilan",
             "CASHBOX": "Sélectionner une caisse",
-            "COST_CENTER": "Sélectionner centre de cout",
-            "COUNTRY": "Sélectionner pays",
+            "COST_CENTER": "Sélectionnez un centre de coûts",
+            "COUNTRY": "Sélectionner le pays",
             "CREDITOR": "Sélectionner un Créditeur",
-            "CREDITOR_GROUP": "Sélectionner Groupe de créditeurs",
+            "CREDITOR_GROUP": "Sélectionner un Groupe créditeur",
```

Comment:
A mon avis, nous devons dire "Group Créditeur" tous en majescule, ou "group créditeur" tous en miniscule.

Reply:
I preferred to use the first formulation "Groupe Créditeur"